.github/scripts/scrape_locales.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\DomCrawler\Crawler;
+
+const URL = 'https://developer.paypal.com/reference/locale-codes/';
+
+$client = HttpClient::create();
+$response = $client->request('GET', URL);
+$html = $response->getContent();
+
+$crawler = new Crawler($html);
+$locales = [];
+
+$crawler->filter('table tbody tr')->each(function (Crawler $row) use (&$locales) {
+    $columns = $row->filter('td');
+    if ($columns->count() < 3) {
+        return;
+    }
+
+    $countryCode = trim($columns->eq(1)->text());
+
+    if (isset($locales[$countryCode])) {
+        return;
+    }
+
+//    $locales[$countryCode][] = [
+//        'region' => trim($columns->eq(0)->text()),
+//        'country_code' => $countryCode,
+//        'priority' => trim($columns->eq(2)->text()),
+//        'locale_code' => trim($columns->eq(3)->text()),
+//        'bcp47_code' => trim($columns->eq(4)->text()),
+//    ];
```

Comment:
I left this in just to have the data available to us in the PR, so we can discuss.

There are many ways to skin this particular cat. We need to decide how fancy we want this locale 'guesser' to be. Since Paypal lists all supported locales by priority, I thought it best to take the highest priority as the fallback.

This means that having an object such as below is overkill.

```json
{
    'AU': {[
        // locale 1,
        ],[
        // locale 2,
    ]}
}```

I am open to suggestions here though.